sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -5176,3 +5177,15 @@ def test_issue_6134():
      0                              0                   \
 """)
     assert upretty(e) == ucode_str
+
+def test_Vector():
+    #To be completed
+    C = CoordSysCartesian('C')
+    i, j, k = C.base_vectors()
+    a, b, c = symbols('a b c')
+    v = a ** 2 * C.i + b * C.j + c * sin(a) * C.k
+    expected = """\
+ 2
+a  n_x + b n_y + c*sin(a) n_z\
```

Comment:
The unit vectors need some better printing. I would suggest using unicode subscripts and either bolding or hatting the vectors.